tests/pytest_framework/functional_tests/logpath/test_flags_catch_all.py

```diff
@@ -42,17 +42,12 @@ def test_flags_catch_all(tc):
     config = tc.new_config()
 
     file_source = config.create_file_source(file_name="input.log")
-    source_group = config.create_statement_group(file_source)
-
     file_destination1 = config.create_file_destination(file_name="output1.log")
```

Comment:
In `file_destination1`, `1` could be removed, as there is no other `file_destination` in this function.

Reply:
Fixed thanks.